src/getThreadPictures.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function getThreadPictures(threadID, offset, limit, callback) {
+    if(!callback) {
```

Comment:
Don't make callback optional, it wouldn't make any sense not to give a callback.